karate-demo/pom.xml

```diff
@@ -22,19 +22,19 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-dependencies</artifactId>
-            <version>${spring.boot.version}</version>
+            <version>2.2.0.RELEASE</version>
```

Comment:
can this be changed in the parent in one place

Reply:
Added spring boot version in parent pom. Any reason why test cases are failing.

Reply:
This is the build error I get -
[ERROR]   DemoTestParallel.testParallel:34 failed features:
demo.headers.content-type: content-type.feature:14 - path: $['content-type'][0], actual: 'application/json', expected: 'charset=UTF-8', reason: not a sub-string